content/blog/v1-39-released.md

```diff
@@ -0,0 +1,55 @@
+---
+title: What's new in Scratch Addons v1.39
+description: Handle asset import conflicts, delete all but one asset, and more.
+date: null # TBD
+author: DNin01
+---
+
+# What's new in Scratch Addons v1.39.0
+
+Plenty of new features have been added in this update to improve your productivity on Scratch.
+
+Let's start with the new addons.
+
+### 📄 Choose what to do with conflicting asset names
```

Comment:
Use the second-level heading as it is for semantics, not for heading font size reasons (also true to later headings).

```suggestion
## 📄 Choose what to do with conflicting asset names
```

Reply:
I have to fix this with the previous posts later. For this post, this can be done now.